### PR TITLE
fix(themes): border-radius doesn't work with radius factor

### LINKED ISSUE #13108: The roundness property does not set correctly the border radius
## Description  
Setting the `roundness` property of the `theme()` function to a maximum value of 1 does not actually set the maximum border radius value for the components themselves. According to this [table](https://www.infragistics.com/products/ignite-ui-angular/angular/components/themes/sass/roundness#baseline-roundness) from the roundness [topic](https://www.infragistics.com/products/ignite-ui-angular/angular/components/themes/sass/roundness) of our documentation the flat button with `roundness` set to `1` should have a `border radius of 20px`. However, this does not happen and if the `roundness` is set to _1_ it appears to have a `border radius of about 4-5 pixels`.

 * igniteui-angular version: ^14.2 ^15.1 ^16.0

## Steps to reproduce  

1. Go to this [sample ](https://stackblitz.com/edit/go5nh8?file=src%2Fstyles.scss).
2. Observe that in the `theme()` function the `roundness` is set to `1`. 

## Result  
The border radius of the flat button for the column hiding in the grid toolbar has a border radius of about 4-5px.

## Expected result  
The border radius of the flat button for the column hiding in the grid toolbar should have a border radius of 20px.

## Additional
In addition, you can see the difference if you uncomment the part in the code where a border radius of 20px is set on the buttons.

## Attachments  
roundness
issue
border
radius

### PR DESCRIPTION
**Pull request by simeonoff**
Closes #13108  

### Additional information (check all that apply):
 - [x] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/core/styles/components/calendar/_calendar-theme.scss [hunk 1]
```diff
@@ -322,11 +322,11 @@
     }
```
Comment (SisIvanova) [inline projects/igniteui-angular/src/lib/core/styles/components/calendar/_calendar-theme.scss:323]: You missed the previous $border-radius on line 321

**Comment by SisIvanova** [comment]
You should also check the $border-radius in avatar and badge themes.

**Comment by simeonoff** [comment]
> You should also check the $border-radius in avatar and badge themes.

As discussed offline, this change is not needed as the avatar and badge components set their border radius based on the shape attribute.

File: projects/igniteui-angular/src/lib/core/styles/components/calendar/_calendar-theme.scss (hunk 1 above)
Comment (simeonoff) [reply projects/igniteui-angular/src/lib/core/styles/components/calendar/_calendar-theme.scss:323]: It should be fixed now.

**Review by SisIvanova — APPROVED** [review]

**Review by SisIvanova — APPROVED** [review]

**Pull request merged**